bridetalk_project/src/components/atoms/buttons/CancelButton.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import "./cancelbutton.css";
+
+export const CancelButton = (props) => {
```

Comment:
Could also do destructuring in props. Just a comment for learning purposes.

`CancelButton = ({ text, handleClick }) => { .... }`